src/cdk/a11y/input-modality/input-modality-detector.spec.ts

```diff
@@ -180,7 +189,7 @@ describe('InputModalityDetector', () => {
   });
 
   it('should ignore mouse events that occur too closely after a touch event', fakeAsync(() => {
-    detector = new InputModalityDetector(platform, ngZone, document);
+    detector = TestBed.inject(InputModalityDetector);
```

Comment:
Nit: should this also be `setupTest();` like in the tests above? Right now it doesn't configure the test bed at all.

<sup>It obviously still works since the configuration persists between specs + we reference lazy-instantiable DI parameters</sup>

Reply:
Ah good catch, I was doing it with `inject` initially and then I changed my mind, but I must've missed it.